### PR TITLE
[IMP] theme_*: adapt `s_company_team_basic`

### PR DESCRIPTION
**Pull request by mano-odoo**
*: artists, bewise, buzzy, notes, yes, zap

This commit adapts the design themes to the new `s_company_team_basic` snippet.

task-4144648
Part of: task-4077427

Community: https://github.com/odoo/odoo/pull/178323

### CODE REVIEW

File: theme_anelusia/views/snippets/s_company_team_basic.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" inherit_id="website.s_company_team_basic">
+    <!-- Team #01 -->
```
Comment (anso-odoo) [inline theme_anelusia/views/snippets/s_company_team_basic.xml:5]: The structure is a bit weird, shouldn't this change be under `<!-- Team #01 - Img -->` (same for other team members)

Comment (anso-odoo) [inline theme_anelusia/views/snippets/s_company_team_basic.xml:5]: Should this change be under `<!-- Team #01 - Img -->` ? Same for other team members

**Review by anso-odoo — APPROVED** [review]
Hey @mano-odoo can you add all the themes you are targeting in the PR description ?

Also a small comment but after that, LGTM

File: theme_anelusia/views/snippets/s_company_team_basic.xml (hunk 1 above)
Comment (mano-odoo) [reply theme_anelusia/views/snippets/s_company_team_basic.xml]: Woops good catch I don't need to repeat the expression, it's could be part of the same xpath 👍

File: theme_anelusia/views/snippets/s_company_team_basic.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" inherit_id="website.s_company_team_basic">
```
Comment (SergeBayet) [inline theme_anelusia/views/snippets/s_company_team_basic.xml:4]: Consistency. Some inherited (see Be wise theme) templates have a name attribute and some not.

**Review by SergeBayet — CHANGES_REQUESTED** [review]
LGTM, some nitpicking however. See below.

**Review by SergeBayet — APPROVED** [review]

**Pull request closed without merging**